guardrails/cli/hub/install.py

```diff
@@ -208,110 +201,22 @@ def install(
         help="Run the command in quiet mode to reduce output verbosity.",
     ),
 ):
-    verbose_printer = console.print
```

Comment:
theres still a install_hub_module( and a few helpers above around ln 133 in this file are they redundant?

Reply:
Yeah, the PR got a bit bigger than I wanted (mainly due to tests) so was going to rip out the older stuff in a Phase 2 PR, as well as refactor `guardrails create` to use the updated installation process

Reply:
sg